example/tsfile/src/main/java/org/apache/iotdb/tsfile/TsFileRead.java

```diff
@@ -39,7 +39,8 @@
  * TsFileWriteWithTSRecord or TsFileWriteWithTablet to generate the test.tsfile first
  */
 public class TsFileRead {
-  private static final String DEVICE1 = "device_1";
+
+  private static final String DEVICE1 = Constant.STORAGE_GROUP + ".device_1";
```

Comment:
Actually it's not necessary to indicate the storage group for TsFile module. The storage group is a concept of IoTDB only. We are supporting to use TsFile module stand alone and this example is for this purpose.

Reply:
This is because in the TsFileWriteWithTablet class, the code write tsfile use root.sg info. if we write use the class and read use TsFileRead class, it will not work.
So in this case, I will remove the storage info in TsFileWriteWithTablet class.